src/app/components/Ad/Amp/__snapshots__/index.test.tsx.snap

```diff
@@ -62,47 +62,46 @@ exports[`AMP Ads Snapshots should correctly render an AMP leaderboard ad 1`] = `
   text-align: right;
 }
 
-@media (min-width: 20rem) and (max-width: 37.4375rem) {
-  .emotion-8 {
+@media (min-width: 20rem) {
+  .emotion-4 {
     font-size: 0.75rem;
     line-height: 1rem;
   }
 }
 
 @media (min-width: 37.5rem) {
-  .emotion-8 {
+  .emotion-4 {
     font-size: 0.75rem;
     line-height: 1rem;
   }
 }
 
-.emotion-8:hover {
+.emotion-4:hover {
   -webkit-text-decoration: underline;
   text-decoration: underline;
 }
 
 <div>
-  <div
-    amp-access="toggles.ads.enabled"
-    amp-access-hide="true"
-    class="emotion-0 emotion-1"
+  <section
+    aria-hidden="true"
+    aria-label="Publicités"
+    class="emotion-0"
+    data-e2e="advertisement"
+    role="region"
```

Comment:
Do we need to be worried about the ordering of the divs/sections in the snapshot - it appears to have changed, but does that matter?